x-pack/plugins/security_solution/scripts/endpoint/resolver_generator_script.ts

```diff
@@ -37,6 +37,23 @@ async function deleteIndices(indices: string[], client: Client) {
   }
 }
 
+async function addUser(esClient: Client) {
+  try {
+    await esClient.transport.request({
+      method: 'POST',
+      path: '_security/user/endpoint_user',
+      body: {
+        password: 'changeme',
```

Comment:
Can we use provided creds by user args instead of hardcode it?

Reply:
f7e83b54cc2a3dfcc4ae3b2a203e1b1da6868010 added option to add a custom user with `superuser` and `kibana_system` roles.